kong/plugins/statsd/log.lua

```diff
@@ -15,6 +16,7 @@ local ipairs = ipairs
 local tonumber = tonumber
 local knode = kong and kong.node or require "kong.pdk.node".new()
 local null = ngx.null
+local lmdb_info = lmdb.get_env_info
```

Comment:
Don't localize this variable.  It makes it harder to recognize what's going on at the call site.

Reply:
localize related problem has been discussed in another PR, So I choose to ignore this comment.